ga4gh/backend.py

```diff
@@ -654,6 +656,82 @@ def featureSetsGenerator(self, request):
             request, dataset.getNumFeatureSets(),
             dataset.getFeatureSetByIndex)
 
+    def rnaQuantificationGenerator(self, request):
+        """
+        Returns a generator over the (rnaQuantification, nextPageToken) pairs
+        defined by the specified request.
+        """
+        dataset = self.getDataRepository().getDataset(request.dataset_id)
+        if len(request.rna_quantification_id) is 0:
```

Comment:
It doesn't matter in this case (0 is a singleton), but it's generally better to use '==' when comparing numbers.